upup/models/cloudup/resources/addons/kops-controller.addons.k8s.io/k8s-1.16.yaml.template

```diff
@@ -187,3 +207,59 @@ subjects:
 - apiGroup: rbac.authorization.k8s.io
   kind: User
   name: system:serviceaccount:kube-system:kops-controller
+
+---
+
+# permits the node access to create a CSR
```

Comment:
What is using these three ClusterRoleBindings?

Reply:
Ugh .. good point on the RoleBinding to kops-controller - we should always have those `events` permissions in kops-controller, at least.

On the ClusterRoleBindings, these are the magic ClusterRoleBindings that (1) allow a kubelet bootstrap token to create a CSR request, and (2) tell kube-controller-manager to auto-approve CSR requests and (3) tell kube-controller-manager to auto approve CSR renewal requests.  Some docs are here: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#approval